velox/functions/lib/window/tests/WindowTestBase.cpp

```diff
@@ -48,6 +49,81 @@ WindowTestBase::QueryInfo WindowTestBase::buildWindowQuery(
   return {op, functionSql, querySql};
 }
 
+namespace {
+void splitOrderBy(
```

Comment:
This essentially is splitting the string using ',' as a delimiter. There is nothing specific to Order by. It might be simpler to just call this function commaSplit and use it.

Also, it might be easier to do substr outside of the function. And change the signature to return a vector of strings instead of sending it as a function parameter.